src/app/store/onboarding/onboarding.slice.ts

```diff
@@ -28,6 +30,9 @@ export const onboardingSlice = createSlice({
     hideSteps(state, action: PayloadAction<boolean>) {
       state.hideSteps = action.payload;
     },
+    skipFundAccount(state, action: PayloadAction<boolean>) {
+      state.skipFundAccount = action.payload;
+    },
```

Comment:
the fn name is the action, so can be a verb/action phrase, but the state itself, I'm inclined to argue it shouldn't be a verb, `hasSkippedFundAccount` or something. 

Not a big one but came to mind.

Reply:
Yep, makes sense. I'll change.